src/dodal/beamlines/b01-1.py

```diff
@@ -0,0 +1,67 @@
+from ophyd_async.core import StaticDirectoryProvider
+from ophyd_async.epics.areadetector import AravisDetector
+from ophyd_async.panda import HDFPanda
+
+from dodal.common.beamlines.beamline_utils import (
+    device_instantiation,
+)
+from dodal.common.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.log import set_beamline as set_log_beamline
+
+# BL = get_beamline_name("BL01C")
+BL = "c01"
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+
+static_directory_provider = StaticDirectoryProvider("/tmp/bluesky_test_static")
+# set_directory_provider(PandASubdirectoryProvider())
+
+
+def panda(
```

Comment:
Is there a panda on this beamline? I don't see it on the beamline synoptic screen, nor is there a soft-ioc for it in kubernetes

Reply:
yes, there is. there is aan argocd migration going on at the moment.

@marcelldls mentioned it shouldn't take too long